src/Camp24Devon/Tickets.elm

```diff
@@ -0,0 +1,238 @@
+    , description = """
+Ticket for 1 Person including: breakfast, lunch, tea & dinners included. Access to castle grounds. No accommodation included.
+
+• [Nearby accommodation options](/venue-and-access)
+• [Coordinate with other attendees](https://discord.gg/QeZDXJrN78)
+"""
+    , image = "/product1.webp"
+    , slots = 1
+    , productId = Product.ticket.singleRoom
+    }
+
+
+accomToTicket : Accommodation -> Ticket
+accomToTicket accom =
+    case accom of
+        Offsite ->
+            offsite
+
+        Campsite ->
+            campingSpot
+
+        Single ->
+            singleRoom
+
+        Double ->
+            doubleRoom
+
+        Group ->
+            groupRoom
+
+
+accomToString : Accommodation -> String
+accomToString accom =
+    case accom of
+        Offsite ->
+            "Offsite"
+
+        Campsite ->
+            "Camping Spot"
+
+        Single ->
+            "Single Room"
+
+        Double ->
+            "Double Room"
+
+        Group ->
+            "Group Room"
+
+
+allAccommodations =
+    [ Offsite, Campsite, Single, Double, Group ]
+
+
+formIncludesAccom : PurchaseForm -> Bool
+formIncludesAccom form =
+    form.accommodationBookings |> List.filter includesAccom |> List.length |> (\c -> c > 0)
+
+
+includesAccom accom =
+    case accom of
+        Offsite ->
+            False
+
+        Campsite ->
+            False
+
+        _ ->
+            True
+
+
+accommodationOptions : AssocList.Dict (Id ProductId) ( Accommodation, Ticket )
+accommodationOptions =
+    allAccommodations
+        |> List.map
+            (\a ->
+                let
+                    t =
+                        accomToTicket a
+                in
+                ( Id.fromString t.productId, ( a, t ) )
+            )
+        |> AssocList.fromList
+
+
+dict : AssocList.Dict (Id ProductId) Ticket
+dict =
+    [ attendanceTicket, campingSpot, singleRoom, doubleRoom, groupRoom ]
+        |> List.map (\t -> ( Id.fromString t.productId, t ))
+        |> AssocList.fromList
+
+
+viewAccom : PurchaseForm -> Accommodation -> Bool -> msg -> msg -> msg -> Price -> Ticket -> Element msg
+viewAccom form accom ticketAvailable onPress removeMsg addMsg price ticket =
+    let
+        selectedCount =
+            form.accommodationBookings |> List.filter ((==) accom) |> List.length
+    in
+    Theme.panel []
+        [ none
+
+        -- , image [ width (px 120) ] { src = ticket.image, description = "Illustration of a camp" }
+        , paragraph [ Font.semiBold, Font.size 20 ] [ text ticket.name ]
+        , MarkdownThemed.renderFull ticket.description
+        , el
+            [ Font.bold, Font.size 36, alignBottom ]
+            (text (Theme.priceText price))
+        , if ticketAvailable then
+            if selectedCount > 0 then
+                Theme.numericField
+                    "Tickets"
+                    selectedCount
+                    (\_ -> removeMsg)
+                    (\_ -> addMsg)
+
+            else
```

Comment:
Is this waitlist trigger configured for our plan this year? i.e. after 80 campfire tickets sold?

Reply:
Yes, it's based on live availability:

https://github.com/elm-camp/website/blob/d91da8dbf57cc46bbbc98d16294bc2fd9e534eb7/src/Frontend.elm#L1260-L1262